[doc] Fix vSphere doc - disktypes changed according to default policy (#8339)
deckhouse/deckhouse: ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT.md

To add SSH keys to user's authorized keys, the `default_user` parameter must be specified in the `/etc/cloud/cloud.cfg` file.

{% alert level="warning" %}

> Deckhouse creates virtual machine disks with the `eagerZeroedThick` type, but the disk type of the created VMs will be changed without notification according to the `VM Storage Policy` configured in vSphere.
> You can read more in [documentation](https://github.com/hashicorp/terraform-provider-vsphere/blob/main/website/docs/r/virtual_machine.html.markdown#virtual-disk-provisioning-policies).

Reviewer: ```suggestion
Deckhouse creates virtual machine disks of the `eagerZeroedThick` type, however, the disk type of the created VMs will be changed without any notice to match the `VM Storage Policy` as configured in vSphere.
You can read more in the [documentation](https://github.com/hashicorp/terraform-provider-vsphere/blob/main/website/docs/r/virtual_machine.html.markdown#virtual-disk-provisioning-policies).
```
